### PR TITLE
Update Error page

### PR DESCRIPTION
**Pull request by rsrelewa**
This PR updated the error pages with new symbols, error messages and overall made things look pleasing to look at. 
Screenshot_2022-04-03_15-21-01

### CODE REVIEW

File: components/theme/sizes.tsx [hunk 1]
```diff
@@ -5,5 +5,5 @@ export const fontSizes = {
   sm: '0.85rem',
   md: '1rem',
   lg: '1.25rem',
-  xl: '1.5rem'
```
Comment (flacial) [inline components/theme/sizes.tsx:8]: I think it would've been better to create another one called `xxl: 3rem` for example.

**Comment by flacial** [comment]
Each component that could be reused must have a [story](https://storybook.js.org/docs/react/get-started/whats-a-story). The error component already has a [story](https://deploy-preview-1637--youthful-thompson-448332.netlify.app/?path=/story/components-error--not-found), you could modify it instead of creating a new one.

**Review by mino323 — APPROVED** [review]

File: components/admin/AdminLayout.tsx [hunk 2]
```diff
@@ -27,9 +28,10 @@ export const AdminLayout: React.FC<GetAppProps & { title?: string }> = ({
 
   if (!isAdmin) {
     return (
-      <Layout title={title}>
-        <h1>You must be admin to access this page</h1>
-      </Layout>
+      <ErrorPage
+        code={403}
```
Comment (anthonykhoa) [inline components/admin/AdminLayout.tsx:32]: Please use the `StatusCode` enum here instead of hardcoding the value for the `code` prop

**Review by SahilKalra98 — APPROVED** [review]

**Comment by flacial** [comment]
The margin between the elements is small, what about giving the image a margin-bottom of `50px` and the `404 Error` a padding-bottom of `27.2px`

And... for the aesthetics: a drop-shadow `filter: drop-shadow(0px 5px 20px hsl(247.9deg, 66.1%, 80.9%));`

image

**Comment by flacial** [comment]
It's hard to read horizontally long text. Perhaps it can be given a max-width of `500px`.

image

**Review by flacial — APPROVED** [review]

**Review by flacial — CHANGES_REQUESTED** [review]
It's mostly about CSS.

Also, I hope you get well and healthy!

**Comment by songz** [comment]
Closing due to inactivity

**Pull request closed without merging**